SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs

```diff
@@ -300,23 +300,60 @@ public static partial class CsmManager
             }
         }
 
-        private static async Task<Site> CreateSite(ResourceGroup resourceGroup, Func<string> nameGenerator)
+        private static async Task<Site> CreateSite(ResourceGroup resourceGroup, Func<string> nameGenerator, string siteKind = null)
+        {
+            var site = new Site(resourceGroup.SubscriptionId, resourceGroup.ResourceGroupName, nameGenerator(), siteKind);
+            site.Kind = siteKind;
+            await resourceGroup.LoadServerFarms(null);
+            ServerFarm serverFarm = new ServerFarm(resourceGroup.SubscriptionId,resourceGroup.ResourceGroupName,Constants.DefaultServerFarmName,
+                resourceGroup.GeoRegion);
+                var csmSiteResponse =
+                    await
+                        csmClient.HttpInvoke(HttpMethod.Put, ArmUriTemplates.Site.Bind(site),
+                            new
+                            {
+                                properties =
+                                    new { serverFarm = serverFarm,
```

Comment:
use a consistent style for anonymous objects. I'd put the opening `{` on a new line